plugins/modules/docker_host_info.py

```diff
@@ -33,6 +33,8 @@
     description:
       - A dictionary of filter values used for selecting containers to list.
       - "For example, C(until: 24h)."
+      - C(label) is a special case of filter which can be a string C(<key>) matching when a label is present, a string
+        C(<key>=<value>) matching when a label has a particular value, or a list of strings C(<key>)/C(<key>=<value).
```

Comment:
Judging from some tests in docker-py, this might also work for exit codes (`exited: [0, 1]`; see https://github.com/docker/docker-py/blob/master/tests/unit/utils_test.py#L457). I wouldn't mention that in the docs without having tested it first, though...

Though when looking at the Docker daemon code, it looks like that's supported: https://github.com/moby/moby/blob/master/daemon/list.go#L247-L259 builds a list of accepted exit codes from all values passed for the `exited` filter, and https://github.com/moby/moby/blob/master/daemon/list.go#L463-L475 then checks containers for exit codes in that list.

Reply:
I haven't studied these modules thoroughly, but I don't think we ever set `all=True` when calling `containers()` meaning we would only see running containers and `exited` would have no effect as a filter. Unless I'm way off.
```python
        try:
            if docker_object == 'containers':
                items = self.client.containers(**filter_arg)
```

Reply:
To your point however when `all=True` a list of integers for `exited` works:
```yaml
- community.docker.docker_host_info:
         containers: true
         containers_filters:
           exited:
             - 137
             - 0
       register: result

     - debug:
         msg: "{{ result.containers }}"
```
partial result
```json
 {
            "Command": "/bin/bash -c 'sleep 1000000000000'",
            "Created": 1624312686,
            "Id": "a5f2a5adcf5a657221cbfb9d1e3612bdb6dc179d85938da46ed0b8c95004615a",
            "Image": "ubuntu",
            "Names": [
                "/crazy_mestorf"
            ],
            "Ports": [],
            "Status": "Exited (137) 22 hours ago"
        },
        {
            "Command": "/bin/bash",
            "Created": 1624311956,
            "Id": "cfdeb1c381ccd8dddf9dbd4eb11a633d6cda55b283d006dc5f9ce5d7b9fc879c",
            "Image": "ubuntu",
            "Names": [
                "/sweet_dewdney"
            ],
            "Ports": [],
            "Status": "Exited (0) 22 hours ago"
        }
```

Reply:
Right, it needs `all=True`, otherwise there's no exit code ;-) I guess we can add support for `all=True` in a follow-up PR, but right now let's not document `exited`.